srcs/lib/libft/ft_putmembe.c

```diff
@@ -0,0 +1,29 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   ft_putmembe.c                                      :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: ffredrik <ffredrik@student.codam.nl>         +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2019/01/09 17:45:14 by ffredrik      #+#    #+#                 */
+/*   Updated: 2019/02/07 15:40:25 by ffredrik      ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "libft.h"
+
+void		ft_putmembe(void *mem, uint64_t num, size_t size)
+{
+	uint8_t	*mems;
+	t_index idx;
+
+	idx = 0;
+	mems = mem;
+
+	while (idx + 1 <= size)
+	{
+		mems[idx] =
+		((num & (0xFFL << (size - (idx + 1)) * 8)) >> (size - (idx + 1)) * 8);
```

Comment:
WHaatt's happening?

Reply:
Oh don't ask.. this makes sure that whatever endian encoding is used by the CPU it will always write Big Endian to the file.. there is just no easy way to write this down comprehensively

Reply:
Can we put that in a norm compliant comment above the function?

Reply:
Will do